test/test-cases/storybook-config/app/.eslintrc

```diff
@@ -0,0 +1,7 @@
+/** THIS FILE IS GENERATED BY SKU, MANUAL CHANGES WILL BE DISCARDED **/
+{
+  "extends": [
+    "/Users/askoufis/code/fep/sku/node_modules/eslint-config-seek/.eslintrc.js",
```

Comment:
You might want to `.gitignore` this file

Reply:
Oh woops, this is a file from another branch. Will fix it in the other PR.